src/elements/Header/_Header.js

```diff
@@ -1,42 +1,98 @@
 import cx from 'classnames'
-import React, { createElement, Component, PropTypes } from 'react'
-
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {
+  getUnhandledProps,
+  iconPropRenderer,
+  imagePropRenderer,
+  useKeyOnly,
+} from '../../utils/propUtils'
+
+function _Header(props) {
+  const {
+    _sdClass, _headerElement,
+    left, center, right, justified, dividing, block, attached, floating,
+    icon, image, children, className,
+  } = props
+
+  const classes = cx(
+    _sdClass, 'ui',
+    useKeyOnly(left, 'left aligned'),
+    useKeyOnly(center, 'center aligned'),
+    useKeyOnly(right, 'right aligned'),
+    useKeyOnly(justified, 'justified'),
+    useKeyOnly(dividing, 'dividing'),
+    useKeyOnly(block, 'block'),
+    useKeyOnly(attached, 'attached'),
+    useKeyOnly(floating, 'floating'),
```

Comment:
The [floating](http://semantic-ui.com/elements/header#floating) option appears as `right floated` and `left floated` html classes.  These would be expressed in the API as:

```js
<Header floated='right' />
<Header floated='left' />
```

Then in className buildup as:

```js
cx(
  useValueAndKey(floated, 'floated'),
)
```